source/GameInterface/Services/MapEvents/MapEventRegistry.cs

```diff
@@ -19,7 +22,9 @@ public override void RegisterAll()
     {
         foreach (var mapEvent in Campaign.Current.MapEventManager.MapEvents)
         {
-            if (RegisterExistingObject(mapEvent.StringId, mapEvent) == false)
+            int counter = 0;
```

Comment:
This will always be 0, please move outside the loop

Reply:
oh that's awkward